tools/codegen/dest/register_dispatch_key.py

```diff
@@ -59,23 +59,154 @@ def __call__(self, f: Union[StructuredNativeFunctions, NativeFunction]) -> List[
         else:
             assert_never(f)
 
-    def gen_structured_class_set_output(self, k: SchemaKind, parent_class: str, generate_super: bool) -> str:
+    def gen_structured(self, g: StructuredNativeFunctions) -> List[str]:
+        if self.dispatch_key == DispatchKey.Meta:
+            assert self.dispatch_key not in g.out.dispatch, \
+                "Do not explicitly specify Meta dispatch key on structured " \
+                "functions, they will be automatically generated for you"
+        elif not is_structured_dispatch_key(self.dispatch_key):
+            return list(mapMaybe(self.gen_unstructured, g.functions()))
```

Comment:
Not new in this PR I know, but might be worth putting out a message here about falling back to unstructured codegen if you put up another edit

Reply:
Would do it in a separate PR, but I'm not sure I understand the suggestion; are you suggesting some sort of warning? The warning would always trigger, since we are intentionally grandfathering some keys to be non structured.

Reply:
Yeah, I'm thinking of people making the assumption that structured is backend-agnostic. You're right though, it'd be super noisy until every key was structured.